torchvision/transforms/transforms.py

```diff
@@ -199,21 +199,21 @@ def forward(self, image):
 
 
 class ToPILImage:
-    """Convert a tensor or an ndarray to PIL Image - this does not scale values.
+    """Converts a tensor or an ndarray to PIL Image
```

Comment:
Seems like python docstring convention suggests to use imperative verb instead of conjugated verb
Source: https://peps.python.org/pep-0257/

>  It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.

Reply:
Okay, but at [here](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html) also, it's written "Applies." There may be more such cases.

Reply:
These are conventions that may be not respected everywhere for a certain reason. In torchvision (cc @NicolasHug ) we want to follow these convention and in the examples where it is not the case we may want to fix that. Any let's not introduce another convention here.